### ISSUE TITLE
load-keys usage of nvdimm-master.blob and tpm.handle

### ISSUE THREAD

**Issue by jah63** [pmem/ndctl#116]
The load-keys sub command will look for the nvdimm-master.blob and tpm.handle in /etc/ndctl/keys.
Currently, it is the users responsibility to create these files.
Wouldn't it be better for the ndctl tool to save these when doing setup-passphrase?

**Comment by djbw** [comment]
Yes, and that was known at the time the patches went in that ndctl would need to eventually add that support. The current interface puts too much onus on the user to manage this manually. The hold up has been a configuration file syntax to define which master keys apply to which DIMMs. The plan has been to lift the config infrastructure out of git, but finding time for that work has thus far been elusive.

**Comment by jah63** [comment]
This issue relates to load-keys recreating a nvdimm-master key that was created manually.  Do you also plan to add code to setup-passphrase to automate the initial creation of the master key(s)?
Thanks.
